client/a8c-for-agencies/sections/marketplace/controller.tsx

```diff
@@ -98,9 +98,7 @@ export const checkoutContext: Callback = ( context, next ) => {
 			<PageViewTracker title="Marketplace > Checkout" path={ context.path } />
 			<Checkout
 				referralBlogId={ referralBlogId }
-				defaultMarketplaceType={
-					referralBlogId ? MARKETPLACE_TYPE_REFERRAL : MARKETPLACE_TYPE_REGULAR
-				}
+				defaultMarketplaceType={ referralBlogId ? MARKETPLACE_TYPE_REFERRAL : undefined }
```

Comment:
The change looks good: if we pass `undefined` to  `defaultMarketplaceType`, it will result in `regular` MarketplaceType at the end. Which is what should happen.